### PR TITLE
applications: nrf5340_audio: Bugfix volume mute

### PR DESCRIPTION
**Pull request by Khuongh**
- Added hw_codec_mute when VCS is disabled
- Added return values for hw_codec_volume decrease and increase

Signed-off-by: Khuong Huynh <khuong.huynh@nordicsemi.no>

### CODE REVIEW

**Review by koffes — APPROVED** [review]
Nice work!

**Review by rick1082 — APPROVED** [review]

File: applications/nrf5340_audio/src/bluetooth/ble_audio_services.c [hunk 1]
```diff
@@ -385,7 +383,7 @@ int ble_vcs_volume_mute(void)
 #elif (CONFIG_BT_VCS)
 	return bt_vcs_mute(vcs);
 #endif /* (CONFIG_BT_VCS_CLIENT) */
-	return -ENXIO;
+	return hw_codec_volume_mute();
 }
 
 int ble_vcs_volume_unmute(void)
```
Comment (erikrobstad) [inline applications/nrf5340_audio/src/bluetooth/ble_audio_services.c:389]: Should hw_codec_volume_unmute() be added to ble_vcs_volume_unmute()?

**Review by erikrobstad — CHANGES_REQUESTED** [review]
Nice catch!

File: applications/nrf5340_audio/src/bluetooth/ble_audio_services.c (hunk 1 above)
Comment (Khuongh) [reply applications/nrf5340_audio/src/bluetooth/ble_audio_services.c:389]: Added this feature

**Review by erikrobstad — APPROVED** [review]

**Review by gWacey — APPROVED** [review]

**Review by alexsven — APPROVED** [review]
Needs to be rebased

**Pull request merged**